ext/webidl/00_webidl.js

```diff
@@ -375,15 +374,13 @@
     return String(V);
   };
 
+  // deno-lint-ignore no-control-regex
+  const IS_ASCII = /^[\x00-\xFF]*$/;
```

Comment:
ASCII usually refers to 0x00 to 0x7F. Maybe `IS_EXTENDED_ASCII` or `IS_LATIN1`?

Reply:
What about  `IS_BYTE_CHAR`

Reply:
Just went with `IS_BYTE_STRING`